sources/model/entity/PendingUser.ts

```diff
@@ -0,0 +1,68 @@
+import {Column, Entity, Index, PrimaryGeneratedColumn} from "typeorm";
+import { Encrypt } from "../../utils/encrypt";
+import { ApiException } from "../../utils/apiException";
+
+@Entity("PendingUser")
```

Comment:
"pending_user"

I'd like all fields in the database to follow a `snake_case` naming convention